sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,154 @@ def __repr__(self):
         return (f"{self.__class__.__name__}("
                 f"{self.linear_stiffness}, {self.nonlinear_stiffness}, {self.pathway}, "
                 f"equilibrium_length={self.equilibrium_length})")
+
+class CoulombKineticFriction(ForceActuator):
+    """Coulomb kinetic friction with Stribeck and viscous effects.
+
+    Explanation
+    ===========
+
+    This represents a Coulomb kinetic friction with the Stribeck and viscous effect,
+    described by the function:
+
+    .. math::
+        F = f_c sign(v) + (f_{max} - f_c) e^{-(v / v_s)^2} + sigma v
+
+    where :math:`f_c` is the Coulomb friction constant, :math:`f_{max}` is the maximum static friction force,
+    :math:`v_s` is the coefficient of sliding friction, :math:`sigma` is the viscous friction constant,
+    and :math:`v` is the relative velocity.
+
+    The default friction force is :math:`F = mu_k N`, where :math:`N` is the normal force.
+    When specified, the actuator includes:
+
+    - Stribeck effect: :math:`(mu_s - mu_k) N e^{-(v / v_s)^2}`
+    - Viscous effect: :math:`sigma v`
+
+    In case we include all effects, the full actuator represents:
+
+    .. math::
+        F = mu_k N sign(v) + (mu_s - mu_k) N e^{-(v / v_s)^2} + sigma v
+
+    Please note that this actuator assumes slip and applies a force opposite to the velocity direction.
+
+    Parameters
+    ==========
+
+    normal_force : Expr
+        The normal force between the surfaces.
+    mu_k : Expr
+        The coefficient of kinetic friction.
+    pathway : PathwayBase
+        The pathway that the actuator follows.
+    v_s : Expr
+        The coefficient of sliding friction (COF).
+    viscous_coefficient : Expr
```

Comment:
You use shorthand for some variables and the full written out word for some. You should be self consistent within the class, choose one way or the other.

Reply:
How about
```
N : Expr
    The normal force between the surfaces.
mu_k : Expr
    The coefficient of kinetic friction.
pathway : PathwayBase
    The pathway that the actuator follows.
v_s : Expr
    The coefficient of sliding friction (COF).
eta : Expr
    The viscous friction coefficient.
mu_s : Expr, optional
    The coefficient of static friction. Defaults to mu_k, meaning the Stribeck effect evaluates to 0 by default.
```